src/sonic_ax_impl/mibs/ietf/rfc3433.py

```diff
@@ -424,6 +425,16 @@ def update_xcvr_dom_data(self):
 
             # skip RJ45 port
```

Comment:
># skip RJ45 port [](http://example.com/codeflow?start=12&length=16)

Better to move comments near the line of `RJ45_PORT_TYPE`.

Reply:
Fixed